courseworks/week5/TianhaoHu/sparse_vector_mul.ipynb

```diff
@@ -0,0 +1,98 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "using LinearAlgebra, SparseArrays"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "my_spv (generic function with 1 method)"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "function my_spv(A::SparseMatrixCSC{T1},b::Vector{T2}) where {T1,T2}\n",
```

Comment:
Well done, but as we have mentioned in the exercise lesson, it may be not wise to use a 1d sparse matrix to represent a full vector.